test/anonymizer.test.js

```diff
@@ -24,13 +24,65 @@ it("test_anonymization", () => {
     const patientIDTag = dicomDict.dict[tagString];
     const patientIDValue = patientIDTag.Value;
 
-    expect(patientIDValue).toEqual(["Fall 3"]);
+    expect(JSON.stringify(patientIDValue)).toEqual(
+        JSON.stringify([{ Alphabetic: "Fall 3" }])
+    );
+    expect(patientIDValue.toString()).toEqual(["Fall 3"].toString());
 
     // when
     cleanTags(dicomDict.dict);
 
     // then
-    expect(patientIDTag.Value).toEqual(["ANON^PATIENT"]);
+    expect(JSON.stringify(patientIDTag.Value)).toEqual(
+        JSON.stringify([{ Alphabetic: "ANON^PATIENT" }])
+    );
+    expect(patientIDTag.Value.toString()).toEqual(["ANON^PATIENT"].toString());
+});
+
+it("test_anonymization_no_change_ref", () => {
+    // given
+    const arrayBuffer = fs.readFileSync("test/sample-sr.dcm").buffer;
+    const dicomDict = DicomMessage.readFile(arrayBuffer);
+
+    // multiple value name
+    const tagInfo = dcmjs.data.DicomMetaDictionary.nameMap["OtherPatientNames"];
+    const tagNumber = tagInfo.tag,
+        tagString = dcmjs.data.Tag.fromPString(tagNumber).toCleanString();
+
+    const otherPatientNamesIDTag = dicomDict.dict[tagString];
+    const otherPatientNamesIDValue = otherPatientNamesIDTag.Value;
+    const otherPatientNamesIDValueJSON = JSON.stringify(
+        otherPatientNamesIDValue
+    );
+
+    expect(JSON.stringify(otherPatientNamesIDValue)).toEqual(
+        JSON.stringify([
+            {
+                Alphabetic: "Doe^John",
+                Ideographic: "Johnny",
+                Phonetic: "Jonny"
+            },
+            { Alphabetic: "Doe^Jane", Ideographic: "Janie", Phonetic: "Jayne" }
+        ])
+    );
+    expect(otherPatientNamesIDValue.toString()).toEqual(
```

Comment:
Thank you for adding tests here for the patient name handling - that is definitely worthwhile having this tested.